### ISSUE TITLE
Plugins are adding files to .devbox/virtenv/jetpack-io-devbox-plugins/ instead of /.devbox/virtenv/<plugin-name>/

### ISSUE THREAD

**Issue by elliotthilaire-ca** [jetpack-io/devbox-plugins#5]
My devbox version is 0.9.1

According to the README.md documents in each plugin the plugins should be writing to:
 - `devbox.d/<plugin-name>/conf.d/`
 - `.devbox/virtenv/<plugin-name>/process-compose.yaml`

Instead the plugins are writing files to:
 - `devbox.d/jetpack-io-devbox-plugins/conf.d/`
 - `.devbox/virtenv/jetpack-io-devbox-plugins/process-compose.yaml` 

This has the side effect that if multiple plugins are being used then files with the same name, like process-compose.yaml,  
are being overwritten by what ever is defined last.

For example:
```
  "include": [
    "github:jetpack-io/devbox-plugins?dir=rabbitmq",
    "github:jetpack-io/devbox-plugins?dir=mongodb",
  ],
```
Results in .devbox/virtenv/jetpack-io-devbox-plugins/process-compose.yaml containing
```
version: "0.5"

processes:
  mongodb:
    command: "mongod --config=$MONGODB_CONFIG --dbpath=$MONGODB_DATA --bind_ip_all"
    availability:
      restart: on_failure
      max_restarts: 5
``` 

And if I swap the order of the plugins:
```
  "include": [
    "github:jetpack-io/devbox-plugins?dir=mongodb",
    "github:jetpack-io/devbox-plugins?dir=rabbitmq",
  ],
```

Results in .devbox/virtenv/jetpack-io-devbox-plugins/process-compose.yaml containing:
```
version: "0.5"

processes:
  rabbitmq:
    command: "rabbitmq-server"
    availability:
      restart: on_failure
      max_restarts: 5
    daemon: true
    shutdown:
      command: "rabbitmqctl shutdown"
  rabbitmq-logs:
    command: "tail -f $RABBITMQ_LOG_BASE/$RABBITMQ_NODENAME@$(hostname -s).log"
    availability:
      restart: "always"
```

**Comment by jay-aye-see-kay** [comment]
Related to https://github.com/jetpack-io/devbox/issues/1848

**Comment by Lagoja** [comment]
PR to fix this in Devbox at https://github.com/jetpack-io/devbox/pull/1897

**Comment by Lagoja** [comment]
Should be fixed in 0.10.0 release
